app/controllers/api/V1/greetings_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class GreetingsController < ApplicationController
+      def index
+        @greeting = Greeting.find(rand(1..5))
```

Comment:
The requirement is to find a random greeting from all the records and not from the first five records. The #index action is now returning a random record from only the first five records.